academy/4-my-own-chain/can-play.md

```diff
@@ -144,15 +144,199 @@ Now you need to implement the answer to the player's query in `grpc_query_can_pl
     }
     ```
 
-6. If all went fine:
+6. If all went well:
 
-    ```go [https://github.com/cosmos/b9-checkers-academy-draft/blob/b53297d8e87e31b1fc7fb839fce527e66a2a0116/x/checkers/keeper/grpc_query_can_play_move.go#L79-L82]
+    ```go [https://github.com/cosmos/b9-checkers-academy-draft/blob/0f56961/x/checkers/keeper/grpc_query_can_play_move.go#L79-L82]
     return &types.QueryCanPlayMoveResponse{
         Possible: true,
         Reason:   "ok",
     }, nil
     ```
 
+## Interact via the CLI
+
+Friendly reminder that the CLI can always inform you about the available commands:
+
+<CodeGroup>
+<CodeGroupItem title="checkers" active>
+
+```sh
+$ checkersd query checkers --help
+```
+
+Which prints:
+
+```
+...
+Available Commands:
+  can-play-move    Query canPlayMove
+...
+```
+
+</CodeGroupItem>
+<CodeGroupItem title="can-play-move">
+
+```sh
+$ checkersd query checkers can-play-move --help
+```
+
+Which prints:
+
+```
+...
+Usage:
+  checkersd query checkers can-play-move [idValue] [player] [fromX] [fromY] [toX] [toY] [flags]
+...
+```
+
+</CodeGroupItem>
+</CodeGroup>
+
+---
+
+You can test this query at any point in a game's life.
+
+<CodeGroup>
+<CodeGroupItem title="no game" active>
+
+```sh
+$ checkersd query checkers can-play-move 2048 red 1 2 2 3
+```
+
+Trying on a game that does not exist returns:
+
+```
+Error: rpc error: code = InvalidArgument desc = game by id not found: 2048: game by id not found: %s: invalid request
+...
+```
+
+Confirm this was an error from the point of view of the executable:
+
+```sh
+$ echo $?
+```
+
+Which prints:
+
+```
+1
+```
+
+There is room to improve the error message. But at least you got an error, as expected.
+
+</CodeGroupItem>
+<CodeGroupItem title="bad color">
+
+```sh
+$ checkersd tx checkers create-game $alice $bob 1000000 --from $alice -y
+$ checkersd query checkers can-play-move 0 white 1 2 2 3
+```
+
+Trying a bad color for the player on a game that exists returns:
```

Comment:
A bad color? Is there a reason it is bad? Maybe go for an adjective that is less of an anthropomorphism.